src/modules/Calculator.js

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```

Comment:
- [ ] I would recommend you rename the containing folder for this file, `components`, instead of `modules` per the requirement.